src/mappings/DAOFactory/mapping.ts

```diff
@@ -0,0 +1,138 @@
+import { Address, BigInt } from '@graphprotocol/graph-ts';
+import {
+  setContractsInfo,
+  setTemplatesInfo,
+} from '../../contractsInfo';
+
+import { Avatar } from '../../types/Controller/Avatar';
+import { DAOFactory, InitialSchemesSet, NewOrg, ProxyCreated, SchemeInstance } from '../../types/DAOFactory/DAOFactory';
+import {
+  AvatarContract, ContractInfo, DAOFactoryContract,
+} from '../../types/schema';
+import { createTemplate, debug, fetchTemplateName, setContractInfo } from '../../utils';
+
+export function getDAOFactoryContract(address: Address): DAOFactoryContract {
+  let daoFactory = DAOFactoryContract.load(address.toHex()) as DAOFactoryContract;
+  if (daoFactory == null) {
+    daoFactory = new DAOFactoryContract(address.toHex());
+    daoFactory.address = address;
+    let daoFactoryContract = DAOFactory.bind(address);
+    daoFactory.packageName = daoFactoryContract.PACKAGE_NAME();
+    daoFactory.app = daoFactoryContract.app();
+    daoFactory.save();
+    setContractsInfo();
+    setTemplatesInfo();
+  }
+  return daoFactory;
+}
+
+export function handleNewOrg(event: NewOrg): void {
+  // Ensure the DAOTrackerContract has been added to the store
+  getDAOFactoryContract(event.address);
+
+  let avatar = event.params._avatar;
+
+  // If the avatar already exists, early out
+  if (AvatarContract.load(avatar.toHex()) != null) {
+    return;
+  }
+
+  let avatarContract = Avatar.bind(avatar);
```

Comment:
does arc/package version is ignored here ?

Reply:
Basically yes. We did not yet decide how we should deal with multi version support in the new subgraph so for now we still have this "version dependency".